src/components/ApiCard.tsx

```diff
@@ -9,29 +9,34 @@ interface Props {
 function ApiCard({ api }: Props) {
   return (
     <Link target='_blank' href={api.url}>
-      <div className='card mx-3 my-3 w-96 transform rounded-lg bg-base-100 p-4 shadow-xl transition-transform hover:scale-105'>
-        <figure>
-          <Image
-            src={api.image ? `/images/${api.image}` : '/images/default.webp'}
-            alt={api.name}
-            width={100}
-            height={100}
-          />
-        </figure>
-        <div className='card-body'>
-          <h2 className='card-title'>
-            {api.name}
-            <div className='badge badge-secondary'>
-              {api.free ? 'free' : 'paid'}
-            </div>
-          </h2>
-          <p>{api.description}</p>
-          <div className='card-actions justify-end'>
-            {api.categories.map((category, index) => (
-              <div key={index} className='badge badge-outline'>
-                {category}
+      <div className='md: mx-3 my-3 flex w-full   transform flex-col rounded-lg bg-base-100 p-4 shadow-xl transition-transform hover:scale-105 sm:w-full sm:flex-col md:flex-row'>
+        <div className='w-full md:w-1/3'>
+          <figure className='lg: mb-4 mr-0 justify-center  pt-5 md:mb-0 md:mr-4 lg:flex'>
+            <Image
+              src={api.image ? `/images/${api.image}` : '/images/default.webp'}
+              alt={api.name}
+              width={100}
+              height={100}
+              className='mx-auto md:mx-0 '
+            />
+          </figure>
+        </div>
+        <div className='md:2/3 w-full'>
+          <div className='card-body'>
+            <h2 className='text-center text-xl font-bold  md:text-left'>
+              {api.name}
+              <div className='badge badge-secondary'>
```

Comment:
The free badge looks just a bit off. Can you make it align with the title?
---
Current Implementation
<img width="96" alt="image" src="https://github.com/gdsc-fetju/api-collection/assets/76874341/5521b30c-21ec-4691-93d6-2fc1f1abac58">
Expected Outcome
<img width="107" alt="image" src="https://github.com/gdsc-fetju/api-collection/assets/76874341/4b18769d-d1bc-4745-aec1-d10c66dd8fae">